python/foxglove-sdk/python/foxglove/channel.py

```diff
@@ -59,9 +59,11 @@ def __init__(
         _channels_by_topic[topic] = self
 
     def __repr__(self) -> str:
-        return (
-            f"Channel(topic='{self.base.topic()}', schema='{self.base.schema_name()}')"
-        )
+        return f"Channel(id='{self.id()}', topic='{self.topic()}', schema='{self.schema_name()}')"
```

Comment:
Maybe we don't need the quotes around the ID?

```suggestion
        return f"Channel(id={self.id()}, topic='{self.topic()}', schema='{self.schema_name()}')"
```